cadc-access-control-identity/src/main/java/ca/nrc/cadc/ac/ACIdentityManager.java

```diff
@@ -133,15 +137,20 @@ public Set<URI> getSecurityMethods() {
 
     @Override
     public Subject validate(Subject subject) throws NotAuthenticatedException {
-        return TokenValidator.validateTokens(subject);
+        Subject sub = TokenValidator.validateTokens(subject);
+        if (!sub.getPrincipals(AuthorizationTokenPrincipal.class).isEmpty()) {
+            StandardIdentityManager sim = new StandardIdentityManager();
```

Comment:
This can fail with a runtime exception if there is no configured OpenID provider; I feel like that is a sane config so that exception probably needs to be caught and logged (debug).